### ISSUE TITLE
Refactor

### ISSUE THREAD

**Issue by sbonner1** [sbonner1/RateMyDrink#51]
Refactor the backend code to be more efficient and readable.

**Comment by jcoady9** [comment]
working on ratemydrink again?

**Comment by sbonner1** [comment]
Yup, I forgot you would get emails about issues. I thought I'd do it as a side project, some of my backend code is ugly.
